style.css

```diff
@@ -1020,3 +468,129 @@
   color: #42526e;
   text-decoration: none;
 }
+
+@media screen and (max-width: 768px) {
+  .port-nav {
+    display: none;
+  }
+
+  .about-nav {
+    display: none;
+  }
+
+  .contact-nav {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .container {
+    background: url(/icons/Header-llustration-desktop@2x.png) no-repeat;
+    background-size: 100% 800px;
+  }
+
+  .headline {
+    background: none;
+  }
+
+  .navItems .logo {
+    display: none;
+  }
+
+  .navItems .bars {
+    display: none;
+  }
+
+  .navItems {
+    display: flex;
+    flex-direction: row;
```

Comment:
Kindly consider the following changes for your headline section:

- [ ] for both headers please add the correct value for (font size and font weight) to match the Figma design.

- [ ] Kindly add the right margin between the social media icons to match the Figma design.

- [ ] Please add the correct margin value between the whole headline section and the works section below it to match the Figma design.

- [ ] I can see that your paragraph text overlaps with the background image so please fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/161352537-186381fc-ab81-4347-a6ed-515781b83b2e.png)  | ![image](https://user-images.githubusercontent.com/68796232/161352522-ed937cc3-6d2c-4127-9f4f-85930425c225.png)  |